deps/exokit-bindings/canvascontext/src/image-context.cc

```diff
@@ -79,9 +81,10 @@ void Image::Load(Local<ArrayBuffer> arrayBuffer, size_t byteOffset, size_t byteL
     this->cbFn.Reset(cbFn);
     this->error = "";
 
-    uv_async_init(uv_default_loop(), &threadAsync, RunInMainThread);
+    threadAsyncHandle = (uv_async_t *)malloc(sizeof(uv_async_t));
```

Comment:
I'm partial to `new`/`delete`, since this is all C++ code. In the future it could be a smart pointer.